TeamCode/src/main/java/org/firstinspires/ftc/teamcode/core/lib/autonomousControl/TargetVelocityData.java

```diff
@@ -0,0 +1,44 @@
+package org.firstinspires.ftc.teamcode.core.lib.autonomousControl;
+//this is done
+public class TargetVelocityData {
+    double XVel;
+    double YVel;
+    double XV_YV_alpha;
+    double heading;
+    double index;
+    TargetVelocityData(double xVelocity, double yVelocity, double alpha,double desiredHeading,int CourseIndex){
+        /*
+        xPosition means the X axis position relative to the field
+        yPosition means the Y axis position relative to the field
+        heading means the orientation of the robot in degrees relative to the field
+         */
+        XVel = xVelocity;
+        YVel = yVelocity;
+        XV_YV_alpha = alpha;
+        heading = desiredHeading;
+        index = CourseIndex;
+    }
+
+    public double getXV(){
+        return XVel;
+    }
+    public double getYV(){
+        return YVel;
+    }
+    public double getH(){return heading;}
```

Comment:
avoid using inline methods

Reply:
that class ended up being removed due to another review, but why should inline methods be avoided in cases such as this? Do they lower readability or is it to maintain a standart convention?